### PR TITLE
Aws lambda working

### PR DESCRIPTION
**Pull request by DMcCallion**
Changed email report to standardise env variables and target /tmp/ so that file creation works on lambda

Lambda function and permissions / roles created

### CODE REVIEW

**Review by rhKashif — APPROVED** [review]

File: report/requirements.txt [hunk 1]
```diff
@@ -6,4 +6,5 @@ psycopg2-binary
 python-dotenv
 reportlab
 streamlit
```
Comment (gawsalya) [inline report/requirements.txt:8]: can we put the v before the x please

**Review by gawsalya — APPROVED** [review]
nice job!

File: report/lambda_function.py [hunk 2]
```diff
@@ -511,7 +511,7 @@ def create_report(df_releases: DataFrame, dashboard_url: str) -> None:
     </html>
     '''
 
-    with open("test.html", "w", encoding='utf-8') as file:
+    with open("/tmp/test.html", "w", encoding='utf-8') as file:
         file.write(template)
```
Comment (rhKashif) [inline report/lambda_function.py:515]: Did you need this `.html` file? I only added it for testing purposes and forgot to remove. I will remove it from next revision if you don't need it. The email report does not require it to work.

Comment (DMcCallion) [reply report/lambda_function.py:515]: Yeah I don't need it, good point

**Pull request merged**